### PR TITLE
Fix change network on mobile

### LINKED ISSUE #6029: Chose Network button dissapears when using mobile
### What happened?

When resizing the screen or using a mobile phone to access Kodadot, the Change Network button disappears, making it only possible to change networks either by manually writing it in the address bar or choosing to browser assets in a specific network. 

### Please reproduce in steps

- Access Kodadot using a mobile phone
- try to change to a different network

### Expected Behavior

The change network button should be visible: 
image

### What browsers are you seeing the problem on?

Desktop Firefox, Mobile Android Chrome, Mobile Firefox

### At which address did you encounter bug?

canary.kodadot.xyz, beta.kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

Talisman

### At which chain did you encounter bug?

Basilisk, Kusama

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by brano-hozza**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 \_\_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #6029 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/ksm/transfer/?target=Ho4kw64mS5HmTCmGN8vCcQAazkrhqkqi7oyVcY64KjEPBcS)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed navigation on mobile UI; a screenshot is best to understand changes for others.
image image

### <samp>🤖 Generated by Copilot at 68ab5dd</samp>

This pull request adds a new feature to allow users to switch between different chains on mobile devices. It introduces a new component `NavbarChainOptions.vue` that renders a list of chains, and modifies the `Navbar.vue` component to use it in a collapsible section. It also updates the `locales/en.json` file with a new translation key for the chain selection prompt.

### <samp>🤖 Generated by Copilot at 68ab5dd</samp>

> _Sing, O Muse, of the skillful coder who devised_
> _a clever component, `NavbarChainOptions`, to display_
> _the various chains that mortal users could select_
> _and switch with ease, by `useChain` hook and store._

### CODE REVIEW

**Comment by exezbcz** [comment]
nice! works for me, good job

File: components/Navbar.vue [hunk 1]
```diff
@@ -161,6 +170,7 @@ import { useIdentityStore } from '@/stores/identity'
 import { BModalConfig } from 'buefy/types/components'
 import type Vue from 'vue'
 import { NeoIcon } from '@kodadot1/brick'
+import { getChainNameByPrefix } from '~~/utils/chain'
```
Comment (vikiival) [inline components/Navbar.vue:173]:
```suggestion
import { getChainNameByPrefix } from '@/utils/chain'
```

**Review by vikiival — COMMENTED** [review]
Otherwise LGTM

**Comment by prury** [comment]
tested on:
- Firefox (resized window)
- Chrome (resized window)
- Android emulator
- Android mobile

File: components/Navbar.vue [hunk 2]
```diff
@@ -79,6 +79,14 @@
         id="NavChainSelect"
         class="navbar-chain custom-navbar-item"
         data-cy="chain-select" />
+
+      <MobileExpandableSection
+        v-if="isMobile"
```
Comment (Jarsen136) [inline components/Navbar.vue:84]: Could you please change the order between `ChainSelectDropdown` and `MobileExpandableSection`? In this way, we could use `v-else` instead.

**Review by Jarsen136 — COMMENTED** [review]
otherwise lgtm

**Review by Jarsen136 — APPROVED** [review]

**Review by preschian — APPROVED** [review]

**Comment by yangwao** [comment]
pay 50 usd
perfect

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 25.15 USD/KSM ~ 1.988 $KSM
🧗 [Ho4kw64mS5HmTCmGN8vCcQAazkrhqkqi7oyVcY64KjEPBcS](https://kusama.subscan.io/account/Ho4kw64mS5HmTCmGN8vCcQAazkrhqkqi7oyVcY64KjEPBcS)
🔗 [0x2f4a67b529c20425bfede6da04827eee190ecd367068ac6e4ea4c3157496236f](https://kusama.subscan.io/extrinsic/0x2f4a67b529c20425bfede6da04827eee190ecd367068ac6e4ea4c3157496236f)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
